exercises/simple-cipher/canonical-data.json

```diff
@@ -16,13 +17,13 @@
           "input": {
             "plaintext": "aaaaaaaaaa"
           },
-          "expected": "cipher.key"
+          "expected": "cipher.key.substring(0, 10)"
```

Comment:
@cmccandless I think this all looks fine and agrees with the discussion had in #1250.  I have another idea that I want to float...  How about:
```JSON
"expected": "cipher.key.substring(0,plaintext.length)"
```

Reply:
Ooh, I like that even better (albeit with a space before `plaintext`)!